pkg/service/node.go

```diff
@@ -218,15 +218,18 @@ func getDeviceInfo(device string) (string, error) {
 }
 
 func makeFS(device string, fsType string) error {
-	// caution, use -F to force creating the filesystem if it doesn't exit. May not be portable for fs other
-	// than ext family
+	// caution, use force flag when creating the filesystem if it doesn't exit.
 	klog.Infof("Mounting device %s, with FS %s", device, fsType)
-	var force string
+
+	var cmd *exec.Cmd
 	if strings.HasPrefix(fsType, "ext") {
-		force = "-F"
+		cmd = exec.Command("mkfs", "-F", "-t", fsType, device)
+	} else if strings.HasPrefix(fsType, "xfs") {
+		cmd = exec.Command("mkfs", "-t", fsType, "-f", device)
+	} else {
+		return errors.New(fsType + " is not supported, only xfs and ext are supported")
 	}
 
-	cmd := exec.Command("mkfs", force, "-t", fsType, device)
 	err := cmd.Run()
```

Comment:
Can you please log stdout+stderr in case of error? Right now `err` says "mkfs failed with exit status 1", which is not much.